addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -8,12 +8,11 @@
             <field name="arch" type="xml">
                 <pivot string="Livechat Support Statistics" sample="1" display_quantity="1">
                     <field name="has_call" type="measure" invisible="1"/>
-                    <field name="call_duration_hour" type="measure" widget="float_time"/>
                     <field name="partner_id" type="row"/>
-                    <field name="percentage_of_calls" type="measure" widget="percentage"/>
```

Comment:
It's not ok to remove those measures, as we need to apply the correct widget. We should probably use the `pivot_measures` context key to select the measures we want.